src/coreclr/tools/aot/ILCompiler.LLVM/CodeGen/LLVMObjectWriter.cs

```diff
@@ -692,12 +695,14 @@ public void EmitSymbolDefinition(int currentOffset)
 
         //System.IO.FileStream _file;
         string _objectFilePath;
+        bool nativeLib;
```

Comment:
Should this be `_nativeLib` for consistency?

Reply:
yes, changed.